assets/src/components/list-items/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export function ListItems( { className = '', isDisc = false, heading, items } ) {
+	const slugify = ( value ) => {
+		if ( 'string' !== typeof value ) {
+			return value;
+		}
+
+		return value
+			.toString()
+			.trim()
+			.toLowerCase()
+			.replace( /\s+/g, '-' )
+			.replace( /[^\w\-]+/g, '-' )
+			.replace( /\-\-+/g, '-' )
+			.replace( /^-+/, '-' )
+			.replace( /-+$/, '-' );
+	};
+
+	if ( isDisc ) {
+		className = classnames( 'list-items--list-style-disc', className );
+	}
+
+	return (
+		<ul className={ classnames( 'list-items', className ) }>
+			{ heading && (
+				<li className="list-items__item">
+					<h4 className="list-items__heading">
+						{ heading }
+					</h4>
+				</li>
+			) }
+			{ items.map( ( item, index ) => {
+				const key = item.label ? slugify( item.label ) + `-${ index }` : index.toString();
+
+				return (
+					<li key={ key } className="list-items__item">
```

Comment:
In light of the comment under `slugify` function, this can be reverted to:
```suggestion
					<li key={ index } className="list-items__item">
```